tests/functional/f/fixme.py

```diff
@@ -20,7 +20,7 @@ def function():
     #todo: no space after hash
 
     # +1: [fixme]
-	# FIXME: this is broken
```

Comment:
I think the functional test should not change here, this is probably a test in itself :)

Reply:
It look like there is still a change here. You can fix this by recreating the commit from scratch. 
```
git reset master # (pylint's master)
git add . -p # Yes for all except fixme.py
git commit -m "Fix column index of FIXME warnings #4246" -m "The column index value at the miscellaneous checker was changed." -m "The grouping of the match variable was affecting the column index used each time (variable note)," -m "so the note variable was removed to fix that." -m "Also, the expected column indexes of the functional test of FIXME warnings were changed.'"
```

Reply:
sorry for that. However, when I use the git add command after I reset master no changes are being added. I can fix the indentation again and the column index at `fixme.txt` as well without recreating the commit.

Reply:
Neverming, fixing it with a suggestion in github was doable :)